chapter_12_organising_test_files.asciidoc

```diff
@@ -561,9 +568,10 @@ saves you from having to use `def..():` and an indented block:
 5
 ----
 
-In our case, we're using it to transform a bit of code that would otherwise be
-executed immediately into a function that we can pass as an argument,
+In our case, we're using it to transform a bit of code - that would otherwise be
+executed immediately - into a function that we can pass as an argument,
 and that can be executed later, and multiple times:
+// CSANAD: I think it's easier to read like this.
```

Comment:
not sure i agree but nbd.

fwiw the preferred syntax for emdashes is `word--nextword`